<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="8" failed="0" ignored="0" total="8" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Test Suite: CIL Test Suite" duration-ms="3087" started-at="2018-03-03T06:52:33Z" finished-at="2018-03-03T06:52:36Z">
    <groups>
    </groups>
    <test name="CIL Test Cases" duration-ms="3087" started-at="2018-03-03T06:52:33Z" finished-at="2018-03-03T06:52:36Z">
      <class name="com.cil.ui.CILTestSuite">
        <test-method status="PASS" signature="beforeSuiteExtentSetup()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="beforeSuiteExtentSetup" is-config="true" duration-ms="175" started-at="2018-03-03T12:22:33Z" finished-at="2018-03-03T12:22:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuiteExtentSetup -->
        <test-method status="FAIL" signature="setUp(java.lang.String)[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="setUp" is-config="true" duration-ms="2985" started-at="2018-03-03T12:22:33Z" finished-at="2018-03-03T12:22:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.common.LoginToApp.loginToApplication(LoginToApp.java:24)
	at com.common.BaseTest.setUp(BaseTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="beforeMethodExtentSetup(java.lang.reflect.Method)[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="beforeMethodExtentSetup" is-config="true" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethodExtentSetup -->
        <test-method status="SKIP" signature="VerifyCILDashboardComponents()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyCILDashboardComponents" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.common.LoginToApp.loginToApplication(LoginToApp.java:24)
	at com.common.BaseTest.setUp(BaseTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyCILDashboardComponents -->
        <test-method status="SKIP" signature="beforeMethodExtentSetup(java.lang.reflect.Method)[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="beforeMethodExtentSetup" is-config="true" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethodExtentSetup -->
        <test-method status="SKIP" signature="VerifyLinksOnCILDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyLinksOnCILDashboard" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.common.LoginToApp.loginToApplication(LoginToApp.java:24)
	at com.common.BaseTest.setUp(BaseTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLinksOnCILDashboard -->
        <test-method status="SKIP" signature="beforeMethodExtentSetup(java.lang.reflect.Method)[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="beforeMethodExtentSetup" is-config="true" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethodExtentSetup -->
        <test-method status="SKIP" signature="VerifyOnGoingProjects()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyOnGoingProjects" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.common.LoginToApp.loginToApplication(LoginToApp.java:24)
	at com.common.BaseTest.setUp(BaseTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyOnGoingProjects -->
        <test-method status="SKIP" signature="VerifyToolsAndTechnologiesOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyToolsAndTechnologiesOnDashboard" duration-ms="0" started-at="2018-03-03T12:22:36Z" depends-on-methods="com.cil.ui.CILTestSuite.VerifyOnGoingProjects" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CILTestSuite.VerifyToolsAndTechnologiesOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CILTestSuite.VerifyToolsAndTechnologiesOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyToolsAndTechnologiesOnDashboard -->
        <test-method status="SKIP" signature="VerifyUpcomingEventsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyUpcomingEventsOnDashboard" duration-ms="0" started-at="2018-03-03T12:22:36Z" depends-on-methods="com.cil.ui.CILTestSuite.VerifyToolsAndTechnologiesOnDashboard" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CILTestSuite.VerifyUpcomingEventsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CILTestSuite.VerifyUpcomingEventsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUpcomingEventsOnDashboard -->
        <test-method status="SKIP" signature="VerifyLabVisitsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyLabVisitsOnDashboard" duration-ms="0" started-at="2018-03-03T12:22:36Z" depends-on-methods="com.cil.ui.CILTestSuite.VerifyUpcomingEventsOnDashboard" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CILTestSuite.VerifyLabVisitsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CILTestSuite.VerifyLabVisitsOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLabVisitsOnDashboard -->
        <test-method status="SKIP" signature="VerifyTrendingIdeasOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyTrendingIdeasOnDashboard" duration-ms="0" started-at="2018-03-03T12:22:36Z" depends-on-methods="com.cil.ui.CILTestSuite.VerifyLabVisitsOnDashboard" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CILTestSuite.VerifyTrendingIdeasOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CILTestSuite.VerifyTrendingIdeasOnDashboard()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTrendingIdeasOnDashboard -->
        <test-method status="SKIP" signature="VerifyOpinionPollSubmit()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="VerifyOpinionPollSubmit" duration-ms="0" started-at="2018-03-03T12:22:36Z" depends-on-methods="com.cil.ui.CILTestSuite.VerifyTrendingIdeasOnDashboard" finished-at="2018-03-03T12:22:36Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CILTestSuite.VerifyOpinionPollSubmit()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CILTestSuite.VerifyOpinionPollSubmit()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyOpinionPollSubmit -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="endReport()[pri:0, instance:com.cil.ui.CILTestSuite@42d80b78]" name="endReport" is-config="true" duration-ms="0" started-at="2018-03-03T12:22:36Z" finished-at="2018-03-03T12:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.cil.ui.CILTestSuite -->
    </test> <!-- CIL Test Cases -->
  </suite> <!-- Test Suite: CIL Test Suite -->
</testng-results>
